pubtools/_quay/operator_pusher.py

```diff
@@ -524,6 +527,22 @@ def build_index_images(self):
         if failed_items:
             return {}
 
+        BuildIndexImageParam = namedtuple(
```

Comment:
I know it's we didn't have a jira for anything like this, but since you started adding new model class here, can you please put it in new module pubtools/_quay/models.py and use dataclasses instead of namedtuple?

Reply:
Done.